R/RcppExports.R

```diff
@@ -14,6 +14,6 @@
 #' @export
 #' 
 fmatch <- function(x, y) {
-    .Call('_fmatch_fmatch', PACKAGE = 'fmatch', x, y)
```

Comment:
@dholstius I did a google search for "roxygen and Rcpp" and wound up here: https://gallery.rcpp.org/articles/documenting-rcpp-packages/. It says

> When using Rcpp in package development, the R functions in RcppExports.R are generated automatically from the source .cpp files. Rcpp also faithfully transcribes comment blocks from .cpp to .R. We can take advantage of this fact to add documentation directly to the .cpp files and have it appear in the package reference manual.

Based on that I looked for differences in this file in the two versions. I noticed that the `PACKAGE` parameter is missing. I wonder if that could be related to our bug.